### PR TITLE
CASMINST-4951: Extend test timeouts to avoid false failures, CASMINST-4961: Improvements to check_static_routes test script

### PR DESCRIPTION
**Pull request by mharding-hpe**
## Summary and Scope

During the most recent csm-1.3 install attempt on surtur, there were false test failures due to test timeouts. This modifies our tests to be less aggressive about timing out.

In addition, problems were observed with manually running the check_static_routes test, and the logic it employed. This pulls in improvements to that test as well.

## Testing

I tested the modified tests on surtur.

## Risks and Mitigations

Low risk -- for the timeout change, the only change is to the timeout value for tests with low timeouts. For the static route test change, almost all of the changes are confined to a single Bash function.

## Pull Request Checklist

- [X] License file intact
- [X] Target branch correct
- [X] Testing is appropriate and complete, if applicable

### CODE REVIEW

File: rpm/cray/csm/sle-15sp2/index.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/:
     - cray-uai-util-2.1.0-1.x86_64
     - craycli-0.56.0-1.x86_64
     - csm-install-workarounds-1.12.1-1.noarch
-    - csm-testing-1.14.28-1.noarch
+    - csm-testing-1.14.29-1.noarch
```
Comment (mharding-hpe) [inline rpm/cray/csm/sle-15sp2/index.yaml:30]:
```suggestion
    - csm-testing-1.14.30-1.noarch
```

**Comment by denniswalker** [comment]
1.14.30-1 already existed. Can you bump to the next version?

**Comment by rustydb** [comment]
This is already updated in main.

**Comment by rustydb** [comment]
https://github.com/Cray-HPE/csm/commit/94c5c4412057b64fc593bfd00444c3fed6d2ed43

**Pull request closed without merging**